cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -526,6 +528,11 @@ define([
                 expect(this.view.$el).toContainText(emptyMessage);
                 expect(this.view.$(SELECTORS.itemView)).not.toExist();
             });
+
+            it('can create a new group configuration', function () {
+                this.view.$('.new-button').click();
```

Comment:
Which thing is being a created? A new experiment group configuration, or a new content group?